src/app/component/MiniCart/MiniCart.component.js

```diff
@@ -39,9 +39,16 @@ class MiniCart extends Component {
         this.handleItemFocus = this.handleItemFocus.bind(this);
     }
 
-    handleItemClick() {
+    handleItemClick(event) {
+        const { products } = this.props;
+        const cartIsEmpty = Object.keys(products).length === 0;
```

Comment:
No need for JavaScript here. It is possible to do with CSS, pointer-events property for example.